### ISSUE TITLE
The method 'api' isn't defined for the class 'ForgotPasswordController'

### ISSUE THREAD

**Issue by lhilton** [nylo-core/laravel_auth_slate#3]
**Note:** This issue comes about after a manual correction noted at the end of issue #2. Please review that in case the details are part of the problem here.

After performing a clean installation as noted in #2, Nylo v5.28.0 and laravel_auth_slate v1.0.25, attempting to `flutter run` produces a build error. The build error is:

```
> flutter run
Launching lib/main.dart on iPhone 15 Pro Max in debug mode...
Running Xcode build...                                                  
Xcode build done.                                            3.6s
Failed to build iOS app
Error (Xcode):
lib/app/controllers/forgot_password_controller.dart:23:33: Error: The
method 'api' isn't defined for the class 'ForgotPasswordController'.


Could not build the application for the simulator.
Error launching application on iPhone 15 Pro Max.
```

**Comment by lhilton** [comment]
I should add that I updated the controllers impacted by this issue and found a patch that worked. For example in the file `login_controller.dart`, I found this line:

```dart
LaravelAuthResponse? laravelAuthResponse = await api<LaravelAuthApiService>((request) => request.login(email, password), context: context);
```

Per the docs, it appears that the below solution should be equivalent and appears to work in my test app:

```dart
LaravelAuthApiService _apiService = LaravelAuthApiService();
// ...
LaravelAuthResponse? laravelAuthResponse = await _apiService.login(email, password);
```

From the docs though it appears that `api` should exist as a global helper function, so this is likely not the proper solution for this slate.

**Comment by agordn52** [comment]
Hi @lhilton,

Thanks for reporting this.

The stub was missing an import `import 'package:nylo_framework/nylo_framework.dart';`.
I've resolved that now in laravel_auth_slate [1.1.1](https://pub.dev/packages/laravel_auth_slate).
